test.js

```diff
@@ -1,5 +1,12 @@
-var t = "hello";
+  var t = "hello";
+var b = null
```

Comment:
Reporter: CHECKSTYLE
Rule: eslint.rules.no-unused-vars
Severity: ERROR
File: /var/lib/openshift/57246f9589f5cf681e000139/app-root/data/jobs/QA-PR-CHECK/workspace/test.js L2
 

&apos;b&apos; is defined but never used (no-unused-vars)

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>